### PR TITLE
add possibility to configure via command line arguments

### PR DESCRIPTION
**Pull request by adamsitnik**
Since I want to run various configurations I need to be able to somehow configure it.

The aspnet/benchmarks allow to pass env vars or console line arguments to the benchmarked app. IMHO console line args are the best here.

### CODE REVIEW

**Comment by adamsitnik** [comment]
Example:

```cmd
-e epoll -t 2 -a true -s true -c true -r true
```

**Review by tmds — APPROVED** [review]

**Comment by tmds** [comment]
I've added you as a collaborator. I assume you can now also merge PRs. Feel free to do that if you want to.

File: test/web/Program.cs [hunk 1]
```diff
@@ -1,41 +1,106 @@
...
+        public bool UseAio { get; set; }
+
+        [Option('c', "dispatch-continuations", Required = false, Default = false, HelpText = "Dispatch Continuations")]
+        public bool DispatchContinuations { get; set; }
+
+        [Option('s', "defer-sends", Required = false, Default = true, HelpText = "Defer Sends")]
+        public bool DeferSends { get; set; }
+
+        [Option('r', "defer-receives", Required = false, Default = true, HelpText = "Defer Receives")]
+        public bool DeferReceives { get; set; }
```
Comment (tmds) [inline test/web/Program.cs:35]: Let's set these defaults to match the default Socket behavior:
dispatch-continuations: true
defer-sends: false
defer-receives: false

**Comment by adamsitnik** [comment]
> I've added you as a collaborator.

Awesome, thanks! I've solved the conflict and changed the default values - merging now

**Pull request merged**